features/core-verify-checksums.feature

```diff
@@ -29,6 +30,7 @@ Feature: Validate checksums for WordPress install
     Then STDERR should be:
       """
       Warning: File doesn't exist: readme.html
+      Warning: File should not exist: readme.html.bak
```

Comment:
Given that it's relatively standard to include extra files in the web root, I was thinking that we'd exclusively check `wp-admin` and `wp-includes` — and ignore the web root entirely. Thoughts?